engines/bastion_katello/test/debs/debs.controller.test.js

```diff
@@ -0,0 +1,102 @@
+describe('Controller: DebsController', function() {
+    var $scope,
+        $location,
+        $controller,
+        dependencies,
+        Deb,
+        Task,
+        Repository,
+        Nutupane;
+
+    beforeEach(module('Bastion.debs', 'Bastion.test-mocks'));
+
+    beforeEach(function() {
+        Nutupane = function() {
+            this.table = {
+                params: {},
+                showColumns: function() {}
+            };
+            this.get = function() {};
+            this.setParams = function (params) {};
+            this.getParams = function (params) { return {}; };
+            this.refresh = function () {};
+            this.getAllSelectedResults = function () {};
+        };
+        Errata = {};
+
+        Task = {
+            registerSearch: function() {},
+            unregisterSearch: function () {}
+        };
+    });
+
+    beforeEach(inject(function(_$controller_, $rootScope, _$location_, MockResource, translateMock) {
+        Deb = MockResource.$new();
+        Repository = MockResource.$new();
+        $scope = $rootScope.$new();
+        $location = _$location_;
+
+        $controller = _$controller_;
+        dependencies = {
+            $scope: $scope,
+            $location: $location,
+            Nutupane: Nutupane,
+            Deb: Deb,
+            Task: Task,
+            Repository: Repository,
+            CurrentOrganization: 'CurrentOrganization',
+            translate: translateMock
+        };
+
+        $controller('DebsController', dependencies);
+    }));
+
+    it('attaches the nutupane table to the scope', function() {
+        expect($scope.table).toBeDefined();
+    });
+
+
+    it('gets a list of deb repositories for the organization', function () {
```

Comment:
BTW, in case you're not aware of how to run the Angular tests in your local dev environment instead of waiting for Jenkins: https://github.com/Katello/katello/blob/master/engines/bastion_katello/README.md

Reply:
Oh right, just remembered that's why I commented them out in the first place :sweat_smile: 
OK, I removed the stuff from the angular tests for now.
If we have to add something to the UI, we should do it in a separate Issue/PR (or wait until React-UI lands :smiling_imp: )